constrain/library/hpwh_sizing.py

```diff
@@ -0,0 +1,34 @@
+from constrain.checklib import RuleCheckBase
+
+
+class HPWH_sizing(RuleCheckBase):
+    points = [
+        "T_amb",
+        "HeatingRate_dx_coil",
+        "HeatingRate_waterheater1",
+        "HeatingRate_waterheater2",
+        "T_amb_parameter",
+    ]
+
+    def hpwh_load(self, data):
+        total_hpwh_load = (
+            data["HeatingRate_dx_coil"]
+            + data["HeatingRate_waterheater1"]
+            + data["HeatingRate_waterheater2"]
+        )
+
+        if data["T_amb"] <= data["T_amb_parameter"] or total_hpwh_load == 0.0:
+            return "untested"
+        elif data["HeatingRate_dx_coil"] / total_hpwh_load >= 1:
+            return True
```

Comment:
I think this is missing something. We want to check what percentage of the load is met at `T_amb_parameter` so I think we need parameter to do that. For instance, are we meeting 50% of the load at 4 deg. C.

Reply:
Thank you Jeremy. Based on your discussion, I updated the logic. I'd appreciate it if you could look into it.